### DISCUSSION TITLE
Parallel running model FAST V8 SUM file locked

### DISCUSSION THREAD

**Discussion by Lateafter** [OpenFAST/openfast#1893] [Q&A]
When I parsim in matlab by FAST v8，coming out with such error ：“FAST_InitializeAll:ED_Init:OpenFOutFile:Cannot open file "E:/w/posst/CertTest/Test25.SFunc.ED.sum". Another program like MS Excel may have locked it for writing.” on ’ShowSimulationManager‘.Anyone save me from it，luv u bro

**Comment by andrew-platt** [comment]
I'm not entirely certain I understand your question.  I don't have enough information to really answer your question as it is stated.

1. From the `CertTest` in your path, I assume some Exactly which subversion of FAST v8 are you running?
2. How are you running this?  Through a script, from the command line, or through Simulink?
3. How is it compiled, or where and when did you download the executable?  
4. Do you actually have that file open someplace?

**Comment by Jqyang632** [comment]
I encountered the same situation as you. Did you solve it, bro? 






,
